src/ui/pages/settings/hooks/useUserInfo.ts

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { pipe, remoteData, taskEither } from '@code-expert/prelude';
+import { UserInfo, getUserInfo } from '@/domain/UserInfo';
+import { Exception } from '@/domain/exception';
+import { useRaceState } from '@/ui/hooks/useRaceState';
+
+export const useUserInfo = () => {
```

Comment:
We can simplify this with the useRemoteData hook that I wrongly removed from the project 🙈